hvantk/commands/hgc_cli.py

```
@@ -0,0 +1,374 @@
+        parent_dir = os.path.dirname(output_path)
+        if parent_dir and not os.path.exists(parent_dir):
+            if create_dirs:
+                os.makedirs(parent_dir, exist_ok=True)
+            else:
+                return False
+        return True
+    except Exception:
+        return False
+
+
+def estimate_resource_requirements(file_paths, operation='combine'):
```

Ruff (0.14.1)

Unused function argument: `operation`

(ARG001)